src/System.Globalization/tests/NumberFormatInfo/NumberFormatInfoCurrentInfo.cs

```diff
@@ -20,46 +21,35 @@ public static IEnumerable<object[]> CurrentInfo_CustomCulture_TestData()
         [MemberData(nameof(CurrentInfo_CustomCulture_TestData))]
         public void CurrentInfo_CustomCulture(CultureInfo newCurrentCulture)
         {
-            CultureInfo oldCurrentCulture = CultureInfo.CurrentCulture;
-            try
+            RemoteInvoke((cultureCode) =>
             {
-                CultureInfo.CurrentCulture = newCurrentCulture;
-                Assert.Same(newCurrentCulture.NumberFormat, NumberFormatInfo.CurrentInfo);
-            }
-            finally
-            {
-                CultureInfo.CurrentCulture = oldCurrentCulture;
-            }
+                CultureInfo newCulture = new CultureInfo(cultureCode);
+                CultureInfo.CurrentCulture = newCulture;
+                Assert.Same(newCulture.NumberFormat, NumberFormatInfo.CurrentInfo);
+                return SuccessExitCode;
+            }, newCurrentCulture.TwoLetterISOLanguageName).Dispose();
```

Comment:
> I originally had it as a local method variable, but that failed for a reason I don't recall.

A local variable outside of the RemoteInvoke lambda?  That won't work because the function name is being marshaled across to the other project, not any of the in-process data associated with the closure.  The lambda passed to RemoteInvoke must not close over any data.